sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -44,6 +44,7 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper wit
       pushDownFilters,
       pushDownAggregates,
       pushDownLimits,
```

Comment:
why do we still keep it?

Reply:
LIMIT should be pushed by `pushDownLimitAndOffset` as well

Reply:
OK